packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx

```diff
@@ -0,0 +1,209 @@
+          options={ this._testOptions }
+        />
+
+        <ComboBox
+          label='Disabled uncontrolled example with defaultSelectedKey:'
+          defaultSelectedKey='D'
+          options={
+            [
+              { key: 'A', text: 'Option a' },
+              { key: 'B', text: 'Option b' },
+              { key: 'C', text: 'Option c' },
+              { key: 'D', text: 'Option d' },
+              { key: 'E', text: 'Option e' },
+              { key: 'F', text: 'Option f' },
+              { key: 'G', text: 'Option g' },
+            ]
+          }
+          disabled={ true }
+        />
+
+        { value ?
+          <ComboBox
+            label='Basic controlled example:'
+            id='Basicdrop5'
+            ariaLabel='Basic ComboBox example'
+            allowFreeform={ true }
+            autoComplete={ true }
+            options={ options }
+            onChanged={ this._onChanged }
+            onResolveOptions={ this._getOptions }
+            value={ value && value }
+          />
+          :
+          <ComboBox
+            selectedKey={ selectedOptionKey && selectedOptionKey }
+            label='Basic controlled example:'
+            id='Basicdrop5'
+            ariaLabel='Basic ComboBox example'
+            allowFreeform={ true }
+            autoComplete={ true }
+            options={ options }
+            onChanged={ this._onChanged }
+            onResolveOptions={ this._getOptions }
+          />
+        }
+
+      </div>
+
+    );
+  }
+
+  @autobind
+  private _getOptions(currentOptions: IComboBoxOption[]): IComboBoxOption[] {
+
+    if (this.state.options.length > 0) {
+      return this.state.options;
+    }
+
+    let newOptions =
+      [
+        { key: 'Header', text: 'Theme Fonts', itemType: SelectableOptionMenuItemType.Header },
+        { key: 'A', text: 'Arial Black', fontFamily: '"Arial Black", "Arial Black_MSFontService", sans-serif' },
+        { key: 'B', text: 'Time New Roman', fontFamily: '"Times New Roman", "Times New Roman_MSFontService", serif' },
+        { key: 'C', text: 'Comic Sans MS', fontFamily: '"Comic Sans MS", "Comic Sans MS_MSFontService", fantasy' },
+        { key: 'C1', text: 'Calibri', fontFamily: 'Calibri, Calibri_MSFontService, sans-serif' },
+        { key: 'divider_2', text: '-', itemType: SelectableOptionMenuItemType.Divider },
+        { key: 'Header1', text: 'Other Options', itemType: SelectableOptionMenuItemType.Header },
+        { key: 'D', text: 'Option d' },
+        { key: 'E', text: 'Option e' },
+        { key: 'F', text: 'Option f' },
+        { key: 'G', text: 'Option g' },
+        { key: 'H', text: 'Option h' },
+        { key: 'I', text: 'Option i' },
+        { key: 'J', text: 'Option j' }
+      ];
+    this.setState({
+      options: newOptions,
+      selectedOptionKey: 'C1',
+      value: null
+    });
+
+    return newOptions;
+  }
+
+  @autobind
+  private _optionsContainsKey(key: string): boolean {

```

Comment:
actually, after working on the example this function is no longer needed and I've removed it